modules/serverless-admin-monitoring-eventing-channel.adoc

```diff
@@ -0,0 +1,28 @@
+[id="serverless-admin-monitoring-eventing-channel_{context}"]
+= Monitoring Knative Eventing channels
+
+You can use the {product-title} monitoring dashboards to view metrics for channels in your cluster.
+
+.Prerequisites
+
+* You have cluster administrator permissions, and access to the *Administrator* perspective in the {product-title} web console.
+* You have installed the {ServerlessOperatorName}, the Knative Eventing component, and an the `KnativeKafka` custom resource.
+* The {product-title} monitoring stack is enabled on your cluster. You can enable monitoring for {ServerlessProductName} during installation by checking the box to *Enable operator recommended cluster monitoring on this namespace* when installing the {ServerlessOperatorName}.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Monitoring* -> *Dashboards*.
+. Select the *Knative Eventing - Channel* dashboard in the *Dashboard* drop-down list.
+. You can now view the following metrics:
+.. For in-memory channels:
+*** Event Count (avg/sec, over 1m window)
```

Comment:
@mikemckiernan The format that you are seeing here is different from those other places on this page because the caption in the interface has changed, to precisely what you see here - `Event Count (avg/sec, over 1m window)`. I expect the other lists to adopt the new format as well, in time.